app/src/app/usePyrogTheme.ts

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+
+import { createMuiTheme, useMediaQuery, Theme } from "@material-ui/core";
+
+declare module "@material-ui/core/styles/createPalette" {
+  interface Palette {
+    badges: {
+      required: string;
+      pending: string;
+    };
+    icons: {
+      table: string;
+      disabled: string;
+      fhir: string;
+      resourceTree: Palette["primary"];
```

Comment:
Or use this type `Palette["primary"]` indeed.

Why not having use `Palette["primary"]` for every color you added to the palette ?

Reply:
I guess you did this to have only one color and not having to call `.main` every time ?

Reply:
Yes